File: sdks/php/src/sdk/src/Documents/Types/InitializeRenderJobRequestTarget.php
<?php

namespace Pogodoc\Documents\Types;

enum InitializeRenderJobRequestTarget: string
{
    case Pdf = "pdf";
    case Html = "html";
    case Docx = "docx";
    case Xlsx = "xlsx";
    case Pptx = "pptx";
    case Png = "png";
    case Jpg = "jpg";
}

File: sdks/ruby/src/client/lib/pogodoc_api_client/documents/types/initialize_render_job_request_target.rb
# frozen_string_literal: true

module PogodocApiClient
  class Documents
    # Type of output to be rendered
    class InitializeRenderJobRequestTarget
      PDF = "pdf"
      HTML = "html"
      DOCX = "docx"
      XLSX = "xlsx"
      PPTX = "pptx"
      PNG = "png"
      JPG = "jpg"
    end
  end
end
